login/service_whitebox_test.go

```diff
@@ -79,6 +84,20 @@ func TestValidOAuthAccessToken(t *testing.T) {
 	assert.Equal(t, identity.Username, claims.Username)
 }
 
+// Can't use test.GenerateToken() because it would introduce a cycle dependency between test and login packages
+func generateToken(ident account.Identity) (string, error) {
```

Comment:
Yes. I moved token.GenerateToken() to token/token_test.go because we actually need it for tests only. But I can't use it in the white box tests in the login package because of cycle dependencies. So, I copied this token generation to this particular tests. The rest of our tests can (and use) the moved function in the test package.